oaknational/Oak-Web-Application
# feat: use next/image in CardImage component, add AspectRatio component

## Description

- Adds `<AspectRatio />` component
- Fixes next/image storybook hack
- Adds AspectRatio to `<CardImage />` component, 1 for consistency and 2. so that when the image loads there's no layout shift
- Users next/image in `<CardImage />`

## Issue(s)

Fixes #207 
Progresses #161 

## How to test

1. Go to {storybook}
2. Click on _______
3. You should see CardImage with correct aspect ratio, and no layout shift when image loads

## Checklist
